### PR TITLE
finish header with out logic ag

### PR DESCRIPTION
**Pull request by DominikReszczynski**

### CODE REVIEW

File: src/components/AppHeader.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const AppHeader = () => {
 					alt='Fischkapp logo'
 				/>
 			</a>
-			<p className='headerConteiner__cardInfo'>{`Cards:${count}`}</p>
+			<p className='headerConteiner__cardInfo'>{`Cards :${count}`}</p>
```
Comment (Sumick) [inline src/components/AppHeader.tsx]: **issue:** You don't have to use BEM when you have CSS modules configured. Just do:

```
<p className={styles.cardInfo}>{`Cards :${count}`}</p>
```

After that, your styles will be scoped to this specific component without the risk of overwriting from other component.

**Review by Sumick — CHANGES_REQUESTED** [review]
Your work looks nice! However, let's consider refactoring the BEM classnames to utilize CSS modules. Since we're already scoping styles with CSS modules, the use of the BEM naming convention may not be necessary. Check `AppLayout` component, and see how it's done. 

More info about CSS modules: https://blog.openreplay.com/using-css-modules-in-react/

File: src/components/AppHeader.tsx (hunk 1 above)
Comment (DominikReszczynski) [reply src/components/AppHeader.tsx]: Now it should be fine. I removed BEM and improved the code, and in the process, I learned something new :)

**Review by Sumick — COMMENTED** [review]
I can't approve this PR as it's already merged ;( Can you please restart task on Nerdbord, and attach some dummy PR with comment inside? I will approve it manually

**Pull request merged**